cli/crates/server/src/bridge/udf.rs

```diff
@@ -396,7 +398,7 @@ pub async fn invoke(
 
     messages.sort_by_key(|(logged_time, _)| *logged_time);
     for (_, message) in messages {
-        bridge_sender.send(message).await.unwrap();
+        bridge_sender.send(message).unwrap();
```

Comment:
unwrap here

Reply:
well yeah like before 🤷

Reply:
I'd use an expect still if we're modifying it anyhow

Reply:
ok